### PR TITLE
Updating /cmds - Main

### PR DESCRIPTION
**Pull request by XilentLVP**
Adding /account in /cmds -> Main
Edit for #663

### CODE REVIEW

**Comment by Specifer** [comment]
Ah cool..

File: pawn/Resources/Callbacks/Others/OnDialogResponse.pwn [hunk 1]
```diff
@@ -297,7 +297,11 @@ deprecated_OnDialogResponse(playerid, dialogid, response, listitem, inputtext[])
                 return 1;
 
             switch(listitem) {
-                case 0: ShowPlayerDialog(playerid, DIALOG_COMMANDS_MAIN, DIALOG_STYLE_MSGBOX, "Main commands", "/help\r\n/stats\r\n/nos\r\n/getstats\r\n/world\r\n/my\r\n/minigames\r\n/jump\r\n/cd", "Back", "Exit");
+
+                case 0: ShowPlayerDialog(playerid, DIALOG_COMMANDS_MAIN, DIALOG_STYLE_MSGBOX, "Main commands", "/account\r\n/help\r\n/stats\r\n/nos\r\n/getstats\r\n/showmessage\r\n/world\r\n/my\r\n/minigames\r\n/jump\r\n/cd", "Back", "Exit");
+=======
```
Comment (RussellLVP) [inline pawn/Resources/Callbacks/Others/OnDialogResponse.pwn]: Would you mind fixing the merge conflict? Lines 301 and 303 roughly do the same, but we should only have one! 😛

**Comment by RussellLVP** [comment]
Ace, thank you for the contribution @XilentLVP!!! One comment on a merge conflict.

**Review by RussellLVP — APPROVED** [review]
Thank you! Approved.

**Pull request merged**